examples/direct_fidelity_estimation.py

```diff
@@ -276,6 +277,24 @@ def _estimate_pauli_traces_general(qubits: List[cirq.Qid],
     return pauli_traces
 
 
+def _estimate_std_devs_clifford(fidelity: float, N: int):
```

Comment:
add return type of Tuple[float, float].  This will help as the doc string does not make it immediately obvious that this is a Tuple returned.

Reply:
Done.